docs/src/modules/components/AppFooter.js

```diff
@@ -165,7 +165,7 @@ function AppFooter(props) {
                 license: (
                   <Link
                     color="inherit"
-                    href="https://github.com/mui-org/material-ui/blob/next/LICENSE"
+                    href={`https://github.com/mui-org/material-ui/tag/v${process.env.LIB_VERSION}/LICENSE`}
```

Comment:
Would `blob/HEAD` be good enough here? The license is very unlikely to change between versions.

Reply:
Not sure to be honest. I don't think the license linked on the page has any legal implication so even if we did move it, it would still be available in the published package.

I'm mostly concerned with older versions of the docs here not maintenance work.